misc/v3-task-endpoint-validator/v3-task-endpoint-validator.go

```diff
@@ -168,12 +176,21 @@ func verifyTaskStats(client *http.Client, taskStatsEndpoint string) error {
 
 	fmt.Printf("Received task stats: %s \n", string(body))
 
-	var taskStats map[string]*types.Stats
+	var taskStats map[string]*types.StatsJSON
 	err = json.Unmarshal(body, &taskStats)
 	if err != nil {
 		return fmt.Errorf("task stats: unable to parse response body: %v", err)
 	}
 
+	if isBridgeNetworkMode {
```

Comment:
curious, do we also want to validate for other network mode or the containerStats.Networks will be blank other than bridge mode?

Reply:
it's blank in other modes